content/spin/v2/manifest-reference-v1.md

```diff
@@ -106,6 +106,7 @@ Each table in the `component` array contains the following fields:
 |                         |            | Table       | * A directory to be made available to the Wasm module at a specific path. This must be a table containing a `source` field for the directory relative to the manifest file, and a `destination` field containing the absolute path at which to make it available. | `{ source = "assets/images", destination = "/pictures" }` |
 | `exclude_files`         | Optional   | Array of strings | Any files or glob patterns that should _not_ be available to the Wasm module at runtime, even though they match a `files` entry. | `[assets/images/test/**/*.*]` |
 | `allowed_http_hosts`    | Optional   | Array of strings | The host names or addresses to which the Wasm module is allowed to send HTTP requests. If the name includes a port, the Wasm module can send requests only to that port; otherwise, the Wasm module can send requests only to the default port for the scheme it uses. The special string `insecure:allow-all` permits the module to send HTTP requests to _any_ host, but is intended for development use only; some deployment environments may decline to honour it. | `["example.com", "localhost:8081"]` |
+| `allowed_outbound_hosts` | Optional  | Array of strings | The addresses to which the Wasm component is allowed to send network requests. Each entry must contain both a name (or IP address) and a port in `name:port` format. For Redis, MySQL, and PostgreSQL, the component may only contact the addresses listed here; for HTTP, the component may also contact hosts in `allowed_http_hosts`. If this field is omitted, Spin 1 components default to allow-all access for backward compatibility; otherwise, no outbound access is permitted except for HTTP. | `["example.com:4567", "127.0.0.1:5000"]` |
```

Comment:
> Otherwise, no outbound access is permitted except for HTTP.

I found this wording confusing. If the `allowed_outbound_hosts` is set, traffic is allowed to anything that the value configured permits. The way it's currently worded sounds like all traffic will be blocked no matter what. Also, http traffic is not openly permitted, it's also blocked unless allowed by `allowed_outbound_hosts` or `allowed_http_hosts`.